onboarding-enabler-nodejs-sample-app/package.json

```diff
@@ -16,5 +16,9 @@
   "dependencies": {
     "@zowe/apiml-onboarding-enabler-nodejs": "file:../onboarding-enabler-nodejs",
     "express": "^4.16.4"
+  },
+  "engines": {
+    "npm": "=7.10.0",
+    "node": "=16.0.0"
```

Comment:
Just wanting to understand it, why a lower version? In any case I think the latest 16.x.x is `16.19.0`

Reply:
Hey Pablo, it's because of the constraint of nodejs version on mainframe. Version above 16.x.x are not supported, so I think that for nodejs enabler we should be stick to this version, but since API catalog UI is bundled in JAR we are not really bound to any specific version.
That's true it can be 16.19.0, I will upgrade it